alf/networks/transformer_networks.py

```diff
@@ -193,4 +201,9 @@ def forward(self, inputs, state=()):
                 query = new_query
```

Comment:
Here memory actually writes the old core, but in the docstring comment the memory writes the new core. This behavior is also different from centralized memory which always writes the new core.

Reply:
But I think we should write the old one? because the new one will be passed as part of input next

Reply:
Good point. Perhaps that was what I thought originally.

Reply:
Should also change the docstring?

Reply:
Already changed.

Reply:
I thought that change was for centralized memory. I mean for centralized memory, we should also put old core in memory?

Reply:
I think putting new core into memory makes more sense for centralized memory.